### ISSUE TITLE
Integrations for #accessibility-docs

### ISSUE THREAD

**Issue by akirk** [Automattic/matrix-bots-wporg#12]
For the `#accessibility-docs` Slack channel on Making WordPress, the following integrations need to be migrated from Slack to Matrix:

 - [ ] Trac Bot
